src/Microsoft.OpenApi.Readers/V3/OpenApiV3VersionService.cs

```diff
@@ -137,11 +138,13 @@ public OpenApiV3VersionService(OpenApiDiagnostic diagnostic)
                         }
                         id = localSegments[3];
                     }
+                    else
+                    {
+                        openApiReference.IsFragrament = true;
+                    }
 
                     return new OpenApiReference
```

Comment:
@MaggieKimani1  @millicentachieng I don't remember which one of you worked on the summary descriptions, can you confirm this change is expected from v1 to v2 please?

Reply:
any update here

Reply:
@Afsalmc we already have a number of people off work for the end of the year celebrations. Thank you for being patient. Full disclosure, this might only be merged next year.

Reply:
Thanks for communicating @baywet

Reply:
@baywet @Afsalmc apologies for the delayed response.
We need to revert the deletion for the summary and description properties in the OpenApiReference object as they are expected in v2 as a result of [this PR](https://github.com/microsoft/OpenAPI.NET/pull/1077)

Reply:
@Afsalmc we can also incorporate this change for us to be at parity with `vnext`, in addition to retaining the summary and description fields
https://github.com/microsoft/OpenAPI.NET/blob/760445cf88cdb6aff00a6ca3a0fd0827f45f91db/src/Microsoft.OpenApi.Readers/V3/OpenApiV3VersionService.cs#L132-L142